environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   - pip
   - numpy
   - scipy
+  - numba
```

Comment:
Where is this used? I thought it was only in the GPU sections?

Reply:
I tried jit to speed up listDistInts but there was no performance increase - removed numba in 5413f26.